### ISSUE TITLE
Spicy food intake

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10152]
**Preferred term label:**
Spicy food intake

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0559575

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Frey syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000026
- label: History of spicy food intake
- parents: [Nutrition history (HP:4000123)](https://hpo.jax.org/app/browse/term/HP:4000123
- pmid: 
- def: A history of high intake of spciy foods with ingrediaents such as capsaicin, pepper, chilli, ginger, garlic, onion, fenugreek, and turmeric.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10152
- synonyms:
